libr/debug/dreg.c

```diff
@@ -171,9 +173,9 @@ R_API int r_debug_reg_list(RDebug *dbg, int type, int size, int rad, const char
 			r_reg_arena_swap (dbg->reg, false);
 			diff = r_reg_get_value (dbg->reg, item);
 			r_reg_arena_swap (dbg->reg, false);
-			delta = value-diff;
-			if (tolower ((ut8)rad) == 'j') {
-				snprintf (strvalue, sizeof (strvalue),"%"PFMT64u, value);
+			delta = value - diff;
+			if (rad == 'j' || rad == 'J') {
```

Comment:
so many checks for rad=='j' || 'J'. add a const bool isJson = (rad == 'j' ||| rad == 'J') on top and just chk for this variable in the rest of places

Reply:
You are right. I think I will just track down all the usage of 'J' and replace all of them with 'j'.